src/app/shared/users.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { UserList } from './users.mock';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UsersService {
+
+  constructor() { }
+
+  getAllUsers() {
+    return UserList;
+  }
+
+  getActiveUsers() {
+    return UserList.filter(function (e) {
```

Comment:
Use `=>` functions instead of anonymous functions. Below as well

Reply:
used the => functions